### DISCUSSION TITLE
How do I install? I dont understand

### DISCUSSION THREAD

**Discussion by NcMasters** [NVlabs/instant-ngp#158] [Q&A]
I really wanna try instant-ngp, but I can't find any video on how to install it, and I don't understand the readme installation.
I tried with visual studio and CMake but I only get errors.

**Comment by Tom94** [comment]
Turns out YouTuber [bycloud](https://www.youtube.com/channel/UCgfe2ooZD3VJPB6aJAnuQng) created [a video tutorial for installing on Windows](https://www.youtube.com/watch?v=kq9xlvz73Rg).

I'm floored this exists at all -- huge kudos to them!

**Reply by NcMasters** [reply]
Yeah https://youtu.be/kq9xlvz73Rg
